transcom/mymove
# B-18165 Add advance details to Move history

## [B-18165](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871851)

## Summary

In my [previous PR for this backlog item](https://github.com/transcom/mymove/pull/11628) I neglected to include tracking changes to advance requests, advance request status, and advance request amount. These changes add those fields to what is being displayed in our MoveHistory component. I also took the liberty of adding a MonetaryFields.js file to out constants directory to house any fields that should be formatted into currency in the Move history.

### How to test

1. Login as a service counselor and find a move that has a PPM shipment
2. From the Move Details view, click edit shipment on a PPM and edit the details (specifically, one the second page where information about an Advance request is displayed)
3. Once you saved, confirm those changes are displayed in the move history
